pkgs/tools/networking/maphosts/default.nix

```diff
@@ -1,12 +1,23 @@
-{ stdenv, lib, bundlerEnv, ruby }:
+{ stdenv, lib, bundlerEnv, ruby, makeWrapper }:
 
-bundlerEnv {
+stdenv.mkDerivation rec {
   name = "maphosts-1.1.1";
 
-  inherit ruby;
-  gemfile = ./Gemfile;
-  lockfile = ./Gemfile.lock;
-  gemset = ./gemset.nix;
+  env = bundlerEnv {
+    name = "maphosts-gems";
+    inherit ruby;
+    gemfile = ./Gemfile;
+    lockfile = ./Gemfile.lock;
+    gemset = ./gemset.nix;
+  };
+
+  phases = ["installPhase"];
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p "$out/bin"
+    makeWrapper "${env}/bin/maphosts" "$out/bin/maphosts"
```

Comment:
In that case you could just use a symlink. makeWrapper is useful if you want to add more runtime dependencies to the program but creates a bit more overhead.